src/queue/multilevel.rs

```diff
@@ -211,6 +211,15 @@ where
             manager: self.manager.clone(),
             local_level0_elapsed_us: self.manager.level0_elapsed_us.local(),
             local_total_elapsed_us: self.manager.total_elapsed_us.local(),
+            tasks_per_level: self.manager.tasks_per_level.clone(),
+            task_execute_duration: self.manager.task_execute_duration.local(),
+            task_wait_duration: self.manager.task_wait_duration.local(),
+            task_poll_duration: self
```

Comment:
To avoid nightly features, I think you can just write it as `array::from_fn(|i| self.manager.task_poll_duration[i].local())`